python/ray/tune/examples/nevergrad_example.py

```diff
@@ -55,8 +55,9 @@ def easy_objective(config, reporter):
         optimizer,
         parameter_names,
         max_concurrent=4,
-        reward_attr="neg_mean_loss")
-    scheduler = AsyncHyperBandScheduler(reward_attr="neg_mean_loss")
+        metric="neg_mean_loss",
+        mode="min")
+    scheduler = AsyncHyperBandScheduler(metric="neg_mean_loss", mode="max")
```

Comment:
the above two should be consistent, and I would do `metric="mean_loss"` and `mode="min"` for both